string strPoseTopicSelected
string strVelocityTopicSelected
---
bool bSuccess
